sdk/basyx/aas/adapter/aasx.py

```diff
@@ -393,8 +412,11 @@ def write_aas(self,
                 try:
                     submodel = submodel_ref.resolve(object_store)
                 except KeyError:
-                    logger.warning("Could not find submodel %s. Skipping it.", str(submodel_ref))
-                    continue
+                    if self.failsafe:
+                        logger.warning("Could not find submodel %s. Skipping it.", str(submodel_ref))
```

Comment:
```suggestion
                        logger.warning(f"Could not find Submodel {submodel_ref}. Skipping it.")
```


Since `Submodel` is a class, I'd capitalize it.